api/SharedWorkerGlobalScope.json

```diff
@@ -0,0 +1,287 @@
+{
+  "api": {
+    "SharedWorkerGlobalScope": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SharedWorkerGlobalScope",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "29"
+          },
+          "firefox_android": {
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "10.6"
+          },
+          "opera_android": {
+            "version_added": "11.5"
+          },
+          "safari": {
+            "version_added": "5"
+          },
+          "safari_ios": {
+            "version_added": "7.1"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "open": {
+        "__compat": {
+          "description": "<code>open</code>",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
```

Comment:
All 3 versions of Chrome should be `false`.

Reply:
👍